src/_assets/scss/style.scss

```diff
@@ -0,0 +1,162 @@
+$color-link: #5398e8;
+$color-text-base: #333;
+$l-header-width: 32vw;
+
+
+html {
+    box-sizing: border-box;
+    -ms-text-size-adjust: 100%;
+    -webkit-text-size-adjust: 100%;
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: inherit;
+}
+
+html,
+body {
+    border: 0;
+    margin: 0;
+    padding: 0;
+    min-height: 100vh;
+}
+
+body {
+    color: $color-text-base;
+    display: flex;
+    font-family: Georgia, Times, serif;
+    font-size: 1.05em;
+    line-height: 1.5;
+}
+
+a {
+    color: $color-link;
+
+    &:hover,
+    &:focus {
+        text-decoration: none;
+    }
+}
+
+h1,
+h2,
+p,
+ol,
+ul {
+    margin: 0;
+}
+
+ul,
+ol {
+    list-style-position: outside;
+    margin-left: 0;
+    padding-left: 0;
+}
+
+h1 {
+    font-size: 2.1em;
```

Comment:
So I guess I was just whipping something up quickly and just stuck with `em`s as it was already there. But we should start to apply an architecture and follow guidelines. I might set that up as part of this PR.